src/server/utilities/liteATIClickTracking/index.ts

```diff
@@ -0,0 +1,92 @@
+export default () => {
+  window.addEventListener('load', () => {
+    document.addEventListener('click', (event: MouseEvent) => {
+      const targetElement = event.target as HTMLElement;
+      if (targetElement?.tagName === 'A') {
+        event.stopPropagation();
+        event.preventDefault();
+
+        const atiURL = targetElement.getAttribute('data-lite-ati-tracking');
+
+        if (atiURL == null) {
+          return;
+        }
+
+        const currentAnchorElement = event.currentTarget as HTMLAnchorElement;
+        const nextPageUrl = currentAnchorElement?.href;
+
+        const {
+          screen: { width, height, colorDepth, pixelDepth },
+          innerWidth,
+          innerHeight,
+        } = window;
+        const now = new Date();
+        const hours = now.getHours();
+        const mins = now.getMinutes();
+        const secs = now.getSeconds();
+
+        // COOKIE SETTINGS
+        const cookieName = 'atuserid';
+        const expires = 397; // expires in 13 months
+        const cookiesForPage = `; ${document.cookie}`;
+        const atUserIdCookie = cookiesForPage.split(`; ${cookieName}=`);
+        let atUserIdValue = null;
+
+        if (atUserIdCookie.length === 2) {
+          const cookieInfo = atUserIdCookie.pop()?.split(';').shift();
+
+          if (cookieInfo) {
+            const decodedCookie = decodeURI(cookieInfo);
+            const user = JSON.parse(decodedCookie);
+            atUserIdValue = user.val;
+          }
+        }
+
+        if (!atUserIdValue && crypto.randomUUID) {
+          atUserIdValue = crypto.randomUUID();
+        }
+
+        const stringifiedCookieValue = JSON.stringify({ val: atUserIdValue });
+        if (atUserIdValue) {
+          document.cookie = `${cookieName}=${stringifiedCookieValue}; path=/; max-age=${expires}; Secure;`;
+        }
+
+        const rValue = [
+          width || 0,
+          height || 0,
+          colorDepth || 0,
+          pixelDepth || 0,
+        ].join('x');
+
+        const reValue = [innerWidth || 0, innerHeight || 0].join('x');
+
+        const hlValue = [hours, mins, secs].join('x');
+
+        let clientSideAtiURL = atiURL
+          .concat('&', 'r=', rValue)
+          .concat('&', 're=', reValue)
+          .concat('&', 'hl=', hlValue);
+
+        if (navigator.language) {
+          clientSideAtiURL = clientSideAtiURL.concat(
+            '&',
+            'lng=',
+            navigator.language,
+          );
+        }
+
+        if (atUserIdValue) {
+          clientSideAtiURL = clientSideAtiURL.concat(
+            '&',
+            'idclient=',
+            atUserIdValue,
+          );
+        }
+
```

Comment:
👋 @Isabella-Mitchell and @shayneahchoon, 

We have been looking into an issue with ATI + Opera Mini ([see Slack thread for context](https://bbc-tpg.slack.com/archives/G03T00R76KX/p1739185663780809)) and in order for traffic to be correctly attributed (e.g. search, direct, social), we need to make sure that we're sending the document.referrer as the `ref` param. 

`ref` also needs to be the last parameter in the ATI URL according to https://github.com/bbc/simorgh/pull/9950

Would you be able to add this here? 🙇‍♀️